met-web/src/components/common/index.tsx

```diff
@@ -175,3 +173,43 @@ export const modalStyle = {
     overflowY: 'scroll',
     color: Palette.text.primary,
 };
+
+export const MetHeader1 = styled(Typography)(() => ({
+    fontSize: '2.3rem',
+    fontWeight: 500,
+    fontFamily: "'BCSans', 'Noto Sans', Verdana, Arial, sans-serif",
+}));
+
+export const MetHeader2 = styled(Typography)(() => ({
+    fontSize: '1.9rem',
+    fontWeight: 500,
+    fontFamily: "'BCSans', 'Noto Sans', Verdana, Arial, sans-serif",
+}));
+
+export const MetHeader3 = styled(Typography)(() => ({
+    fontSize: '1.5rem',
+    fontWeight: 500,
+    fontFamily: "'BCSans', 'Noto Sans', Verdana, Arial, sans-serif",
+}));
+
+export const MetHeader4 = styled(Typography)(() => ({
+    fontSize: '1.3rem',
+    fontWeight: 500,
+    fontFamily: "'BCSans', 'Noto Sans', Verdana, Arial, sans-serif",
+}));
+
+export const MetBody = styled(Typography)(() => ({
+    fontSize: '16px',
+    fontWeight: 500,
+    fontFamily: "'BCSans', 'Noto Sans', Verdana, Arial, sans-serif",
+}));
+
+export const ModalSubtitle = ({
+    children,
+    ...rest
+}: {
+    children: JSX.Element[] | JSX.Element | string;
+    [prop: string]: unknown;
+}) => {
+    return <Typography variant={'subtitle1'}>{children}</Typography>;
+};
```

Comment:
=
![6d406103-c094-4950-99d8-058c0fa88636](https://user-images.githubusercontent.com/91914654/187517632-32b76253-66c1-41c2-90d5-c8872d21201d.png)
Have all of these been added (including paragraph, small text, link, maybe the link is already done in theme)

Reply:
yes Link is already configured and met label as well that's why I only focused on header and body

Reply:
paragraph and small text?